google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Bucket.java

```diff
@@ -968,11 +975,12 @@ public Blob create(String blob, byte[] content, BlobTargetOption... options) {
    * @return a complete blob information
    * @throws StorageException upon failure
    */
-  public Blob create(String blob, InputStream content, BlobWriteOption... options) {
+  public Blob create(String blob, InputStream content, BlobTargetOption... options)
+      throws IOException {
     BlobInfo blobInfo = BlobInfo.newBuilder(BlobId.of(getName(), blob)).build();
-    Tuple<BlobInfo, Storage.BlobWriteOption[]> write =
-        BlobWriteOption.toWriteOptions(blobInfo, options);
-    return storage.create(write.x(), content, write.y());
+    Tuple<BlobInfo, Storage.BlobTargetOption[]> write =
+        BlobTargetOption.toTargetOptions(blobInfo, options);
```

Comment:
Again confirm the implementation details of `BlobWriteOption.toWriteOptions` & `BlobTargetOption.toTargetOptions`... then open this change in a sperate PR for storage client only.

Reply:
You may include other changes for storage clients